<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapView</class>
 <widget class="QWidget" name="MapView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="yAxisComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yAxisMaxSpinBox">
         <property name="minimum">
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum">
          <double>65535.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Min Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yAxisMinSpinBox">
         <property name="minimum">
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum">
          <double>65535.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Num Labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yAxisNumLabelsSpinBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="yAxisResetButton">
         <property name="text">
          <string>Auto Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="zAxisComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Max Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zAxisMaxSpinBox">
           <property name="minimum">
            <double>-65535.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65535.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Min Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zAxisMinSpinBox">
           <property name="minimum">
            <double>-65535.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65535.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Num Labels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zAxisNumLabelsSpinBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="zAxisResetButton">
           <property name="text">
            <string>Auto Scale</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="EMap" name="widget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QComboBox" name="xAxisComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Max Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xAxisMaxSpinBox">
           <property name="minimum">
            <double>-65535.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65535.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Min Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xAxisMinSpinBox">
           <property name="minimum">
            <double>-65535.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65535.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Num Labels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xAxisNumLabelsSpinBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="xAxisResetButton">
           <property name="text">
            <string>Auto Scale</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EMap</class>
   <extends>QWidget</extends>
   <header>emap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
